packages/vertica-nodejs/test/integration/client/timezone-tests.js

```diff
@@ -10,22 +10,29 @@ var date = new Date()
 const pool = new helper.vertica.Pool()
 const suite = new helper.Suite()
 
+/*
+* NOTE: These tests right now are not very useful. We are not parsing timestamps into date objects, 
```

Comment:
Does it make sense to create a JIRA ticket for this?

Reply:
I have a jira for long term type parsing fix, I think it may make sense to add this as a comment onto that jira